apps/sasquatch/src/projectDependency/res/layout/activity_user_document_detail.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
where is this file used?

Reply:
@guperrot this file is used for user document detail in UserDocumentDetailActivity.java

Reply:
Is there any difference with app document view or is it copy paste? If it's the same we don't 2 code bases.

Reply:
ping

Reply:
@guperrot yes we already saw that there are two similar activities, will merge them into one

Reply:
In a new PR: https://github.com/Microsoft/AppCenter-SDK-Android/pull/1078